### ISSUE TITLE
firewalld: ModuleNotFoundError: No module named 'dbus.mainloop.pyqt5'

### ISSUE THREAD

**Issue by jurobystricky** [clearlinux/distribution#1101]
Seems firewalld (firewall-applet in particular) has some missing dependencies issues.
To reproduce:

<pre>
$ sudo swupd bundle-add firewalld
Loading required manifests...
Downloading packs for:
 - firewalld
        ...100%
Finishing packs extraction...
No extra files need to be downloaded
Installing bundle(s) files...
        ...100%
Calling post-update helper scripts
Successfully installed 1 bundle
</pre>

Then execute the following:
<pre>
$ firewall-applet
Traceback (most recent call last):
  File "/usr/bin/firewall-applet", line 31, in <module>
    from dbus.mainloop.pyqt5 import DBusQtMainLoop
ModuleNotFoundError: No module named 'dbus.mainloop.pyqt5'

</pre>

CLR build 30680

**Comment by sunnyflunk** [comment]
Looks like `PyQt5` doesn't have the full dbus support builtin (and therefore missing `/usr/lib/python3.7/site-packages/dbus/mainloop/pyqt5.so`)

I think it needs dev files for dbus-python (and dbus, but that might be there already) in the build.

**Comment by jurobystricky** [comment]
Yes, I believe this is a PyQt5 package issue, firewall-applet is not the culprit here.

**Comment by puneetse** [comment]
It looks like you fixed this shortly after, but I'm getting another missing module now `PyQT5.sip`

**Comment by fkru** [comment]
I was hit by the above-mentioned issue today with firewall-applet 0.7.2 und python-qt5 version 5.13.2. I presume a fix has to come from the python side.

**Comment by SporadicThought** [comment]
Issuing the following command resolves the issue, allowing the firewall-applet to run:
`sudo pip3 install --upgrade PyQt5`
However, issuing pip commands as root may break other packages.

I suppose one solution would be to impose a script using pipenv or virtualenv to launch a compatible version of Python including the appropriate version of PyQt5.

NOTE: I have yet to get the firewall-applet to appear in CL's default Gnome top-panel. However, it may work with a different DE like xfce.
